crates/sncast/src/helpers/interactive.rs

```diff
@@ -0,0 +1,199 @@
+use crate::helpers::config::get_global_config_path;
+use anyhow::{anyhow, Context, Result};
+use camino::Utf8PathBuf;
+use configuration::search_config_upwards_relative_to;
+use dialoguer::theme::ColorfulTheme;
+use dialoguer::Select;
+use std::env::current_dir;
+use std::fs::File;
+use std::io::{Read, Write};
+use std::path::Path;
+use toml_edit::{DocumentMut, Item, Table, Value};
+
+pub fn prompt_to_add_account_as_default(account: &str) -> Result<()> {
+    let mut options = Vec::new();
+
+    if let Some(option) = get_global_config_path().ok().map(|global_path| {
+        format!(
+            "Yes, global default account ({}).",
+            to_tilde_path(global_path.as_str())
+        )
+    }) {
+        options.push(option);
+    }
+
+    if let Some(option) = current_dir()
+        .ok()
+        .and_then(|current_path| Utf8PathBuf::from_path_buf(current_path.clone()).ok())
+        .and_then(|current_path_utf8| search_config_upwards_relative_to(&current_path_utf8).ok())
+        .map(|local_path| {
+            format!(
+                "Yes, local default account ({}).",
+                to_tilde_path(local_path.as_str())
+            )
+        })
+    {
+        options.push(option);
+    }
+
+    options.push("No".to_string());
+
+    let selection = Select::with_theme(&ColorfulTheme::default())
+        .with_prompt("Do you want to make this account default?")
+        .items(&options)
```

Comment:
Wasn't it supposed to allow only one option either local or global, not both?

Reply:
So in the end it was decided to show user his local config if there is one

Reply:
I think both make more sense